configure.ac

```diff
@@ -88,10 +88,23 @@ dnl ######################################################################
 dnl Checks for programs.
 dnl ######################################################################
 
+AC_PROG_GREP
+AC_PROG_EGREP
 AC_PROG_CC
 AC_PROG_MKDIR_P
 AC_EXEEXT
 
+AS_CASE([${target_os}],
+[darwin*], [
+    AC_CHECK_PROG([BREW], [brew], [brew])
+    AC_MSG_CHECKING([for OS X Homebrew])
+    AS_IF([test "x$BREW" = "xbrew"], [
+            have_brew=yes
+            AC_MSG_RESULT([OS X Homebrew detected])
```

Comment:
Seems like we should have a "Homebrew not detected." I haven't used macports in a while but that would be the other reasonable alternative right?

Reply:
@craigcomstock In this case, the only reason why we care about brew, is because it can give us the path to various libraries like OpenSSL etc. I haven't heard of too many people using macports, I'd guess the other, most likely "alternative" (IMO), is no package manager, i.e. finding the libraries in some standard system paths, or by manually supplying them as arguments to configure.

Reply:
Right on. Last time I worked with another developer who used macOS they used macports. Still an else saying homebrew is required would be good.

Reply:
@craigcomstock but it isn't required? It's just one more way / location to look for the libraries. You can specify them via command line arguments or have the libraries in whatever is the default location(s) on the system(?).

Reply:
Yeah, maybe just a suggestion: "No Homebrew detected. You will need to provide command line arguments LDFLAGS/CPPFLAGS to provide locations for required libraries: rsync, openssl, ..."